### PR TITLE
[#4] Generate missing subrules with length specified

### PR DESCRIPTION
**Pull request by vbmacher**
The following now should work:

```
instruction = "JMP": line(5)     ignore8(8) 000 ignore16(16);

%%

"%s %d" = instruction line ignore8 ignore16;
```

(no exceptions)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -58,7 +58,7 @@
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-source-plugin</artifactId>
-                        <version>3.2.0</version>
```
Comment (sulir) [inline pom.xml:61]: This could be in a separate commit, but never mind.

**Review by sulir — APPROVED** [review]
One disadvantage is that typos may become unnoticed. For example, in
```
instr = add(8);
addd = ...;
```
the `add` rule will be auto-generated and `addd` will be ignored without any warning.

However, the proposed feature is useful, so I am merging it. Detection of unused rule definitions (`addd = ...` in our example) could be handy in the future to prevent the mentioned problem.

**Comment by vbmacher** [comment]
I fully agree with the detection of the unused rules. Thanks for creating the ticket!
